python/tvm/meta_schedule/tune.py

```diff
@@ -392,6 +394,21 @@ def tune_tir(
         The cost model to use.
     measure_callbacks : Optional[List[MeasureCallback]]
         The callbacks used during tuning.
+    space : Optional[FnSpaceGenerator]
+        The space generator to use.
+    blocks : Optional[List[str]]
+        A list of block names to tune. If provided, other blocks
```

Comment:
Nit, `A list of block names specifying blocks to be tuned. Note that if the list is not None, blocks outside this list will not be tuned and space can not be given.`